src/formatDate.js

```diff
@@ -50,7 +50,38 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const prevDelim = fromFormat.pop();
```

Comment:
What does mean `delim`? Is this `delimeter`? Don't use shorthand for naming. And it should name `saparator` I think.